scripts/run_e2e_tests.sh

```diff
@@ -70,7 +70,7 @@ wait_for_service IMAGES 127.0.0.1 3001
 echo ""
 echo "> Running cypress tests"
 
-npx cypress run ${CYPRESS_RECORD} --env OC_ENV=$OC_ENV
+npx cypress run ${CYPRESS_RECORD} --env OC_ENV=$OC_ENV --spec "test/cypress/integration/${CYPRESS_TEST_FILES}"
```

Comment:
I don't get the `--spec` here, aren't we passing `--spec` directly in the commands from `package.json`?

Reply:
You mean commands here right; 

https://github.com/opencollective/opencollective-frontend/blob/4bf570a7e236ebe4d5bc629a118534b8379577cb/package.json#L150-L153

But those commands aren't used here nor anywhere else. They are just there for people to execute parts of the E2E tests I think. 🤔

Reply:
Do these commands still work?

Reply:
Yes they do. Maybe there's some value in these since the users can run parts of the e2e tests by running `npm run test:e2e:0` etc as mentioned in the docs; https://github.com/opencollective/opencollective-frontend/blob/main/docs/e2e.md#3-frontend-cypress. Personally I never ran these commands. If you think they are not useful we could remove them as well. 🤔

Reply:
I'm not using them either (usually go with `npm run cypress:open`), but am not sure. If everything works then it's all good for me!